packages/api/src/ontology/Definition.ts

```diff
@@ -118,26 +122,31 @@ export interface ValidPropertyTypes {
   stringTimeseries: any;
 
   attachment: any;
-  geopoint: any;
-  geoshape: any;
+  geopoint: GeoJSON.Point;
+  geoshape: GeoJSON.Geometry;
```

Comment:
Do you want to allow GeometryCollections? I'm also really interested at comparing BSON size/decode speed for geojson stuff across the wire.

Reply:
I believe GeometryCollections are supported but Feature and Feature collections are not